public/app/features/alerting/unified/Analytics.ts

```diff
@@ -22,18 +21,14 @@ export const LogMessages = {
   unknownMessageFromError: 'unknown messageFromError',
 };
 
-// logInfo from '@grafana/runtime' should be used, but it doesn't handle Grafana JS Agent correctly
-export function logInfo(message: string, context: Record<string, string | number> = {}) {
-  if (config.grafanaJavascriptAgent.enabled) {
-    faro.api.pushLog([message], {
-      level: GrafanaLogLevel.INFO,
-      context: { ...context, module: 'Alerting' },
-    });
-  }
+const alertingLogger = createMonitoringLogger('features.alerting', { module: 'Alerting' });
+
+export function logInfo(message: string, context?: Record<string, string>) {
```

Comment:
Should this be rename to `logAlertingInfo` since it uses the alerting logger specifically?

Reply:
Up to you. I kept the original name to reduce number of changes (the logic is the same with additional `source` param sent in context).

Reply:
I've appending some changes :)  https://github.com/grafana/grafana/pull/80075/commits/2dac8798806a1deaec91d6a83de6e6524f0b3770